### ISSUE TITLE
Shift information on multiple food groups can extend off screen

### ISSUE THREAD

**Issue by UndeadZeratul** [squeek502/SpiceOfLife#70]
Spice of Life v1.3.0
Forge v1448
Apple Core v1.1.0

At least on Auto GUI size it did.  On Large, it was fine.  I bet it's probably a vanilla issue with the tooltip rendering, but I figured I'd let you know what I found.

Auto-Sized GUI:
![First Image](http://i.imgur.com/tUesH05.png)

Large-Sized GUI:
![Second Image](http://i.imgur.com/nr5JSpn.png)

**Comment by squeek502** [comment]
Saw this as well while testing. Not sure what to do about it, but I'll try to find a solution.

**Comment by UndeadZeratul** [comment]
Yeah, I assume it's a vanilla thing, and it has an easy work-around, so it's not that big of a deal, but it seems to be due to the large amount of characters in each line on the tooltip.  Maybe have some way to shorten it?
